TYPES.md

```diff
@@ -37,6 +37,7 @@ export default factory({ type: 'keyValue', /* other options */ })
 | `add` | add an instance of your resource | `add(<instance>)` | `{ type: '@trampss/ADD_TODOS', payload: <instance> }` |
 | `update` | update an existing instance of your resource | `update(<instance>)` | `{ type: '@trampss/UPDATE_TODOS', payload: <instance> }` |
 | `addOrUpdate` | update an existing instance of your resource, or add it if not found | `addOrUpdate(<instance>)` | `{ type: '@trampss/ADD_OR_UPDATE_TODOS', payload: <instance> }` |
+| `replace` | replace an existing instance of your resource | `replace(<instance>)` | `{ type: '@trampss/REPLACE_TODOS', payload: <instance> }` |
```

Comment:
Maybe you can add why this is different to `update`

Reply:
:clap: